apis/python/CMakeLists.txt

```diff
@@ -74,4 +74,7 @@ endif()
 install(TARGETS ${VSPY_TARGET_NAME} LIBRARY DESTINATION .)
 
 # Install libtiledb so that we can repackage it with the extension
-install(IMPORTED_RUNTIME_ARTIFACTS TileDB::tiledb_shared DESTINATION lib)
+if (NOT TILEDB_PATH)
```

Comment:
This will not have any effect on the `find_package`: https://github.com/TileDB-Inc/TileDB-Vector-Search/blob/57426618e9ecc688d5025c07d85028e152c88d5e/apis/python/CMakeLists.txt#L30

Reply:
`find_package` will follow `TileDB_DIR` if set, so we should probably use `if (DEFINED CACHE{TileDB_DIR})` here instead.

Reply:
Changed